helper/calcOrbSpec.jl

```diff
@@ -24,6 +24,7 @@ include(folderpath_OWCF*"misc/temp_n_dens.jl") # Load 'analytical' thermal speci
     calcOrbSpec(-||-; o_interp_length=500, thermal_temp=3.0, thermal_dens=1.0e19, prompt_gamma_energy=nothing, debug=false)
 
 Calculate the expected diagnostic spectrum of one orbit. The inputs are as follows
+- analyticCalc - boolean variable for fully analytical spectrum estimation. By default it is set to false for now.
```

Comment:
Just a comment that 'for now' should be removed, since I believe it should be set to false by default in later versions as well